src/models/constructor-items/sub-blocks.ts

```diff
@@ -188,12 +188,18 @@ export interface LayoutItemProps extends ClassNameProps, CardLayoutProps, Analyt
     icon?: PositionedIcon;
 }
 
-export interface ImageCardProps extends CardBaseProps, Pick<ContentBlockProps, 'title' | 'text'> {
+export interface ImageCardProps
+    extends CardBaseProps,
+        CardLayoutProps,
+        Omit<ContentBlockProps, 'colSizes' | 'centered'> {
     image: ImageProps;
     enableImageBorderRadius?: boolean;
     margins?: ImageCardMargins;
     direction?: ImageCardDirection;
     backgroundColor?: string;
+    url?: string;
```

Comment:
```
link: Pick<LinkProps, 'url' | `urlTitle` | `target`> 
```

Reply:
I use these props to have similar interface with other cards, for example 
- https://github.com/gravity-ui/page-constructor/blob/main/src/sub-blocks/BasicCard/schema.ts
- https://github.com/gravity-ui/page-constructor/blob/main/src/sub-blocks/BackgroundCard/schema.ts